public/app/features/alerting/unified/utils/rule-form.ts

```diff
@@ -167,6 +184,23 @@ export function rulerRuleToFormValues(ruleWithLocation: RuleWithLocation): RuleF
   if (isGrafanaRulesSource(ruleSourceName)) {
     if (isGrafanaRulerRule(rule)) {
       const ga = rule.grafana_alert;
+      const contactPoint: ContactPoint | undefined = ga.notification_settings
```

Comment:
Same as above. Maybe we should have two complementary functions which converts the notification settings object

Reply:
done!